object_labels:
  - 7
  - 10
  - 15
  - 18
  - 19
  - 23
  - 24
  - 27
  - 30
  - 31
  - 33
  - 35
  - 36
  - 37
  - 39
  - 41
  - 43
  - 44
  - 47
  - 50
  - 55
  - 57
  - 58
  - 62
  - 64
  - 65
  - 67
  - 69
  - 71
  - 74
  - 75
  - 81
  - 82
  - 89
  - 92
  - 97
  - 98
  - 100
  - 108
  - 110
  - 111
  - 112
  - 115
  - 117
  - 119
  - 124
  - 125
  - 127
  - 129
  - 131
  - 137
  - 138
  - 139
  - 141
  - 142
  - 143
  - 144
  - 147
  - 148
  - 149
